### PR TITLE
Add fixes for modelExtent

### PR DESCRIPTION
**Pull request by hnn0003**
Change modelExtents to be part of the 2d views & make optionally undefined. Should not ever be set to { }.

### CODE REVIEW

File: packages/saved-views-client/src/models/savedViews/View.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ export interface ViewITwinDrawing extends ViewITwin2d {
     [qx: number, qy: number, qz: number, ax: number],
     [qx: number, qy: number, qz: number, ax: number],
   ];
-  modelExtents: [
+  modelExtents?: [
```
Comment (roluk) [inline packages/saved-views-client/src/models/savedViews/View.ts:88]: Don't forget to add a changelog entry to CHANGELOG.md

**Review by roluk — CHANGES_REQUESTED** [review]

File: packages/saved-views-client/src/models/savedViews/View.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (hnn0003) [reply packages/saved-views-client/src/models/savedViews/View.ts:88]: Updated this to have SHapeProps.transform optional too. I think that was the only other thing that needed updating, right @sarah-pontier ?

Comment (hnn0003) [reply packages/saved-views-client/src/models/savedViews/View.ts:88]: @roluk , added a changelog entry.

[the remark this answers is not in the document]
Comment (sarah-pontier) [reply packages/saved-views-client/src/models/savedViews/View.ts:88]: Great! That was the only other change to optional. There's also a few fixes for handling color transformation, but that can be handled in a separate pr

**Review by sarah-pontier — APPROVED** [review]

**Pull request merged**